samples/bluetooth/central_iso/src/main.c

```diff
@@ -0,0 +1,244 @@
+/* main.c - Application main entry point */
+
+/*
+ * Copyright (c) 2015-2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/types.h>
+#include <stddef.h>
+#include <errno.h>
+#include <zephyr.h>
+#include <sys/printk.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/hci.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/uuid.h>
+#include <bluetooth/gatt.h>
+#include <bluetooth/iso.h>
+#include <sys/byteorder.h>
+
+#define MAX_ISO_APP_DATA (CONFIG_BT_ISO_TX_MTU - BT_ISO_CHAN_SEND_RESERVE)
+
+static void start_scan(void);
+
+static struct bt_conn *default_conn;
+static struct k_delayed_work iso_send_work;
+static struct bt_iso_chan iso_chan;
+NET_BUF_POOL_FIXED_DEFINE(tx_pool, 1, CONFIG_BT_ISO_TX_MTU, NULL);
+
+/**
+ * @brief Send ISO data on timeout
+ *
+ * This will send an increasing amount of ISO data, starting from 1 octet.
+ *
+ * First iteration : 0x00
+ * Second iteration: 0x00 0x01
+ * Third iteration : 0x00 0x01 0x02
+ *
+ * And so on, until it wraps around the configured ISO TX MTU (CONFIG_BT_ISO_TX_MTU)
+ *
+ * @param work Pointer to the work structure
+ */
+static void iso_timer_timeout(struct k_work *work)
+{
+	int ret;
+	static uint8_t buf_data[MAX_ISO_APP_DATA];
+	static bool data_initialized;
+	struct net_buf *buf;
+	static size_t len_to_send = 1;
+
+	if (!data_initialized) {
+		for (int i = 0; i < ARRAY_SIZE(buf_data); i++) {
+			buf_data[i] = (uint8_t)i;
+		}
+	}
+
+	buf = net_buf_alloc(&tx_pool, K_FOREVER);
+	net_buf_reserve(buf, BT_ISO_CHAN_SEND_RESERVE);
+
+	net_buf_add_mem(buf, buf_data, len_to_send);
+
+	ret = bt_iso_chan_send(&iso_chan, buf);
+
+	if (ret < 0) {
+		printk("Failed to send ISO data (%d)\n", ret);
+	}
+
+	k_delayed_work_submit(&iso_send_work, K_MSEC(1000));
```

Comment:
This should probably match the intervar of the QoS which is on usec not msec, 1000 usec is quite agressive actually are you intending this to measure throughtput of the controller? I guess for pure throghtput it might be better to leave rtn as 0 or 1 then.

Reply:
That wasn't the intention. I agree that the ISO interval might be too low, and could/should be increased. I'll update this PR with that tomorrow. 

@Vudentz Do you have any suggestions regarding the ACL connection parameters and if they should be set to something besides the default?